input/intro-notes/StructureDefinition-au-practitioner-intro.md

```diff
@@ -1,16 +1,50 @@
+
 - See [Ahpra Data Guidance](guidance.html#ahpra-data-guidance) for detailed guidance on representing Ahpra-sourced data.
-- A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification` 
+- A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification`
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
 - This profile supports the practitioner's gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
-  - *Man or male* may be represented by sending the SNOMED CT code 446151000124109\|Identifies as male gender\|
-  - *Woman or female* may be represented by sending the SNOMED CT code 446141000124107\|Identifies as female gender\|
-  - *Non-binary* may be represented by sending the SNOMED CT code 33791000087105\|Identifies as nonbinary gender\|
-  - *[I/They] use a different term (please specify)*  may be represented by sending only text and no code
-  - *Prefer not to answer* may be represented by sending the code "asked-declined"
-  - *Not stated or inadequately described* may be represented by the code "unknown"
+  - _Man or male_ may be represented by sending the SNOMED CT code 446151000124109\|Identifies as male gender\|
+  - _Woman or female_ may be represented by sending the SNOMED CT code 446141000124107\|Identifies as female gender\|
+  - _Non-binary_ may be represented by sending the SNOMED CT code 33791000087105\|Identifies as nonbinary gender\|
+  - _[I/They] use a different term (please specify)_ may be represented by sending only text and no code
+  - _Prefer not to answer_ may be represented by sending the code "asked-declined"
+  - _Not stated or inadequately described_ may be represented by the code "unknown"
   - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
-- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
\ No newline at end of file
+- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
+
+**Person Recorded Sex Or Gender implementation guidance**
```

Comment:
Apply the four comments on au-patient-intro.md

Reply:
Done.